### DISCUSSION TITLE
Stop streaming music from the commandline

### DISCUSSION THREAD

**Discussion by Groovylein** [librespot-org/librespot#830] [General]
Hello,
is there a possibility to stop the current playing song from the command line?

Regards
Groovylein

**Comment by ashthespy** [comment]
No, `librespot` (the binary) is primarily a Spotify Connect implementation. Commands are hence meant to arrive to it via Spotify's servers.
You could whip up a simple wrapper to uses `curl` to make a request to the [Spotify web API](https://developer.spotify.com/documentation/web-api/). Or go more brute force and can kill and restart the service ;-) 

`librespot` the library can be developed into a full daemon with local control -- look at [spotifyd](https://github.com/Spotifyd/spotifyd)

**Comment by ashthespy** [comment]
You probably want to continue this discussion at https://github.com/librespot-org/librespot/discussions/756 instead.
